### PR TITLE
Add galley for community conversion and analysis projects

### LINKED ISSUE #80: Add Community Conversion Catalogue
Add new page showing conversion pipelines actually in practice used by labs, including

NeuroConv Catalogue: https://neuroconv.readthedocs.io/en/main/catalogue/catalogue.html

and things like

Ecephys in practice with MatNWB and PyNWB: https://dervinism.github.io/bristol-neuroscience-data-guide/tutorials/Bristol%20GIN%20for%20Silicon%20Probe%20Data.html

Ophys in practice with MatNWB and PyNWB: https://dervinism.github.io/bristol-neuroscience-data-guide/tutorials/Bristol%20GIN%20for%20Calcium%20Imaging%20Data.html

and the others from that site for imaging and icephys.

Those community examples still need to be vetted for Best Practices and proper choice/use of neurodata types, but it would still be very useful to acknowledge and point to somewhere in the NWB docs

cc: @dervinism for all the hard work putting the tutorials together

### PR DESCRIPTION
**Pull request by oruebel**
Fix #80

### CODE REVIEW

**Comment by oruebel** [comment]
@CodyCBakerPhD @bendichter @rly here a start for the community gallery we discussed. I added the two items that were listed in the issue but I'm sure there are more items that we should add before merging the page. Please have a look and add missing items.

File: docs/source/community_gallery/community_gallery.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
+.. _community-gallery:
+
+***************************
+Community Gallery
+***************************
+
+This page is a collection of community conversion and analysis projects cataloged here as a convenience reference for NWB users. This list is not a comprehensive list and many of the projects and resources are built and supported by other groups, and are in active development. If you would like to contribute a project or resources, please see the instructions :nwb-overview-src:`here <>`.
```
Comment (CodyCBakerPhD) [inline docs/source/community_gallery/community_gallery.rst:7]:
```suggestion
This page is a collection of community conversion and analysis projects cataloged here as a convenient reference for NWB users. This list is not comprehensive and many of the projects and resources are built and supported by other groups, and are in active development. If you would like to contribute a project or resources, please see the instructions :nwb-overview-src:`here <>`.
```

Minor rewording

File: docs/source/community_gallery/community_gallery.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Data Analysis and Reuse
+-----------------------
+
+.. image:: figures/dandi_example_notebooks.png
+    :class: align-left, no-scaled-link
+    :width: 100
+
+
+The :ref:`analysistools-dandi` (Distributed Archives for Neurophysiology Data Integration) maintains a `collection of example notebooks <https://github.com/dandi/example-notebooks>`_ associated with datasets, conference tools, or more generally notebooks that illustrate the use of data on DANDI.
```
Comment (CodyCBakerPhD) [inline docs/source/community_gallery/community_gallery.rst:43]:
```suggestion
:ref:`analysistools-dandi` (Distributed Archives for Neurophysiology Data Integration) maintains a `collection of example notebooks <https://github.com/dandi/example-notebooks>`_ associated with datasets, conference tools, or more generally notebooks that illustrate the use of data on DANDI.
```

In speaking, I (and I think other people too) normally refer to either 'The DANDI archive' or `DANDI` as a singular entity reference

**Comment by CodyCBakerPhD** [comment]
Minor nit-picks but otherwise looks like the start of something great!

When the IBL team makes their NWB tutorial public we can also add a link here

**Comment by oruebel** [comment]
Here how it looks on RTD https://nwb-overview--83.org.readthedocs.build/en/83/community_gallery/community_gallery.html

**Comment by oruebel** [comment]
I also add then OpenScope DataBook to the page.

**Comment by oruebel** [comment]
@rly @CodyCBakerPhD @bendichter are there any other items we should add on the page before we merge?

**Comment by CodyCBakerPhD** [comment]
DANDI notebooks was the only other one I knew about, cool to see OpenScope as well

**Review by bendichter — APPROVED** [review]

**Pull request merged**